### ISSUE TITLE
Missing setAutoCommit method

### ISSUE THREAD

**Issue by gilfrade** [tariqdaouda/rabaDB#1]
In RabaConnection source you have a comment that its possible to disable autocommit but i do not see it there...

**Comment by tariqdaouda** [comment]
You're right I must have removed it because it was not that useful. Do you need to debug something? You can frame the part that you want to  debug with the functions enableQueryPrint(bool), and enableDebug(bool) of RabaConnection.
